README.md

```diff
@@ -1,2 +1,56 @@
-# ktor-cli
-Command-line tool for creating Ktor projects
+# Ktor CLI
+Ktor CLI is a command-line tool that brings the capability to create, run, and interact with your [Ktor](https://ktor.io/) application from the command line.
```

Comment:
What does interaction mean? Right now it can generate a project and run an application

Reply:
It's for the future. Will remove.